api/openshift.deploy.yml

```diff
@@ -41,9 +41,9 @@ parameters:
   - name: CPU_REQUEST
     value: 30m
   - name: MEMORY_REQUEST
-    value: 150Mi
+    value: 200Mi
   - name: MEMORY_LIMIT
-    value: 1Gi
+    value: 2.2Gi
```

Comment:
I slightly increase to 2.2Gi for MEMORY_LIMIT and also 200Mi for MEMORY_REQUEST. Hope it is acceptable for you. @basilv